stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/MessageInputView.kt

```diff
@@ -678,6 +741,8 @@ public class MessageInputView : ConstraintLayout {
                         maxMessageLengthExceeded = isMessageTooLong()
                     )
 
+                    inputContainsLinks = !canSendLinks && messageText.containsLinks()
```

Comment:
This condition might be a bit weird - maybe the `inputContainsLinks` boolean can be named something else but IDK what it would be.

Worst case - can you add some comments here to explain what this does and why/how

Reply:
This is a good point, on second look the naming confusing for me as well.

I rephrased this as bit and added some comments, however we might adopt a Kotlin like descriptive naming convention and call the value `inputContainsLinksButNotAllowed`. Verbose to a fault but very descriptive.